README.md

```diff
@@ -1,20 +1,52 @@
 # Cloud Enablement Testsuite - ce-testsuite
 
+#### Requirements
+ - Java 7 or higher.
+ - Maven 3 or higher.
+ 
+ 
 #### Before starting
-Make sure you have the ce-arq dependencies available in your local maven repository. If you don't have it yet just clone the project and install it:
+As a first step, make sure you have the ce-arq dependency available in your local maven repository. If you don't have it yet, just clone the project and install it:
 
 ```sh
 $ git clone https://github.com/jboss-openshift/ce-arq.git
-
 $ cd ce-arq
-
 $ mvn clean install
 ```
 
+The above commands will generate the required ce-arq dependency and install it on your local maven repository.
+
+The ce-arq project, in turn, depends on arquillian-cube project. At the moment, we are using the SNAPHOST version, which means you will have to build it locally in order to satisfy this dependency.
+To build it, please follow the steps below:
+
+```sh
+git clone https://github.com/arquillian/arquillian-cube.git
+cd arquillian-cube
+mvn clean install -DskipTests
+```
+
+PS: The step above might not be necessary in the future, since we will use released versions once all the changes we need are made available in Arquillian Cube.
+
+
+#### Useful System Properties configurations
+
+Sometimes, one might want to run more than one test at the same time. This can be accomplished with the following System Property:
```

Comment:
I feel that an explanatory phrase between the first and the second sentence is missing here. Why that system property is necessary?